app/controllers/AMIable.scala

```diff
@@ -56,12 +56,15 @@ class AMIable @Inject()(override val amiableConfigProvider: AmiableConfigProvide
           totalInstancesCount = instances.length,
           PrismLogic.instancesAmisAgePercentiles(instancesWithAmis)
         )
+        allSSAs = ssa :: PrismLogic.instanceSSAs(instances)
```

Comment:
@adamnfish Because we need the count for the requested SSA. (Top group of AMI pills).

The instance SSAs are usually 'full' SSAs with stack, stage and app values. On the other hand the requested SSA is usually a 'filtered' SSA with potentially `None` value for stage and app, which we consider like matching all instances/AMIs.
In ordered to get the instance count for the requested SSA I am adding it to the list.
An other solution could be to compute all possible values for SSAs, including SSAs with empty values (matching all instances/AMIs) but I found this solution would be adding a lot of complexity.